pydaikin/daikin_airbase.py

```diff
@@ -241,7 +239,15 @@ async def set_zone(self, zone_id, key, value):
         }
 
         if self.support_zone_temperature:
+            params.update({"lztemp_c": self.values["lztemp_c"]})
```

Comment:
Isn't this the magic that is needed?

Reply:
this was just something i realised was missing from the old code

the magic is converting back to query string rather than params